modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,139 @@
+
+* After Argo CD is installed, Red Hat Single Sign On (SSO) is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Red Hat SSO installation after the operator is installed which will result in removing the Red Hat SSO deployment from the `openshift-gitops` namespace.
+Additionally, for any new Argo CD instance you can enable Red Hat SSO by setting the flag to true; when the flag is not present, Red Hat SSO will be disabled by default.
+
+* The default Argo CD instance installed by the OpenShift GitOps operator as well as accompanying controllers can now be made to run on the cluster's infrastructure nodes by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
+. Label the existing nodes. In the terminal run the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
+If required, you can also apply taints and isolate the workloads on Infrastructure nodes and prevent other workloads from scheduling on these nodes. Run the following command:
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add `runOnInfra` toggle in the `GitOpsService` Custom Resource. Run the following command:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Add `tolerations` to the `GitOpsService` Custom Resource if taints have been added to the nodes, for example:
++
+[source,yaml]
+----
+  spec:
+    runOnInfra: true
+    tolerations:
+    - effect: NoSchedule
+      key: infra
+      value: reserved
+    - effect: NoExecute
+      key: infra
+      value: reserved
+----
+. Verify that the workloads in the `openshift-gitops` namespace are now scheduled on infrastructure nodes by viewing `Pods -> Pod details` for any pod in the console UI.
+
+[NOTE]
+====
+Any manually added `nodeSelectors` and `tolerations` added to the default Argo CD Custom Resource will be overwritten by toggle and `tolerations` in the `GitOpsService Custom Resource`.
+====
+* Internal communications in Argo CD can now be secured using `TLS` and OpenShift cluster certificates. Argo CD routes can now leverage OpenShift cluster certificates in addition to using external certificate managers such as cert-manager.
+* Gain insights into the GitOps environments directy from the DevConsole using the improved environment details view in DevConsole for OCP 4.9.
+* You will now have access to custom health checks in Argo CD for `DeploymentConfig` resources, Route resources, and `OLM` operators.
+* The GitOps Operator will now conform the naming conventions recommended by the latest operator-sdk by introducing the following changes:
```

Comment:
The GitOps Operator now conforms to the
There are two by..., by..., in the sentence. You can drop 'by introducing the following changes' and add colon after SDK.
sdk/SDK when in doubt about a usage, do a quick search in the existing docs.